packages/wells/src/client/api/wells.ts

```diff
@@ -0,0 +1,178 @@
+import { Asset, AssetsAPI, IdEither } from '@cognite/sdk';
+import { Well } from '../model/Well';
+import { WellHeadLocation } from '../model/WellHeadLocation';
+import { geospatialClient, SpatialRelationshipNameDTO } from './utils';
+
+export class Wells extends AssetsAPI {
+  /**
+   * converts asset metadata into a wellheadlocation
+   *
+   * ```js
+   * var wells: WellHeadLocation = Wells.mapToWellHeadLocation(asset);
+   * ```
+   *
+   * @param asset
+   */
+  static mapToWellHeadLocation = (asset: Asset): WellHeadLocation => {
+    // @ts-ignore
+    return <WellHeadLocation>{
+      // @ts-ignore
+      crs: asset.metadata['crs'],
+      // @ts-ignore
+      x: asset.metadata['x'],
+      // @ts-ignore
+      y: asset.metadata['y'],
+    };
+  };
+
+  /**
+   * converts an asset array into a well array
+   * ```js
+   * var wells: Well[] = Wells.mapToWell(asset);
+   * ```
+   * @param assets
+   */
+  static mapToWell = (assets: Asset[]): Well[] => {
+    return assets.map(asset => {
+      return <Well>(<unknown>{
+        name: asset.name,
+        wellHeadLocation: Wells.mapToWellHeadLocation(asset),
```

Comment:
the well's id from CDF should also be added to the Well object if it's been fetched from CDF